adafruit_display_shapes/roundrect.py

```diff
@@ -157,13 +173,13 @@ def fill(self, color):
             self._palette.make_opaque(2)
 
     @property
-    def outline(self):
+    def outline(self) -> int:
```

Comment:
this one's return and the one in the argument in it's setter can be `Optional[int]` as well